### PR TITLE
Update wsfc-disaster-recovery-through-forced-quorum-sql-server.md

### PR DESCRIPTION
**Pull request by 0x7FFFFFFFFFFFFFFF**
Fix staled links.

### CODE REVIEW

File: docs/sql-server/failover-clusters/windows/wsfc-disaster-recovery-through-forced-quorum-sql-server.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ ms.author: mathoma
 > [!WARNING]  
 >  The user should be well-informed on the concepts and interactions of Windows Server Failover Clustering, WSFC Quorum Models, [!INCLUDE[ssNoVersion](../../../includes/ssnoversion-md.md)], and the environment's specific deployment configuration.  
 >   
->  For more information, see:  [Windows Server Failover Clustering (WSFC) with SQL Server](https://msdn.microsoft.com/library/hh270278\(v=SQL.110\).aspx), [WSFC Quorum Modes and Voting Configuration (SQL Server)](https://msdn.microsoft.com/library/hh270280\(v=SQL.110\).aspx)  
+>  For more information, see:  [Windows Server Failover Clustering (WSFC) with SQL Server](https://docs.microsoft.com/en-us/sql/sql-server/failover-clusters/windows/windows-server-failover-clustering-wsfc-with-sql-server?view=sql-server-ver16), [WSFC Quorum Modes and Voting Configuration (SQL Server)](https://docs.microsoft.com/en-us/sql/sql-server/failover-clusters/windows/wsfc-quorum-modes-and-voting-configuration-sql-server?view=sql-server-ver16)
```
Comment (Jak-MS) [inline docs/sql-server/failover-clusters/windows/wsfc-disaster-recovery-through-forced-quorum-sql-server.md:40]:
```suggestion
>  For more information, see:  [Windows Server Failover Clustering (WSFC) with SQL Server](windows-server-failover-clustering-wsfc-with-sql-server.md?view=sql-server-ver16), [WSFC Quorum Modes and Voting Configuration (SQL Server)](wsfc-quorum-modes-and-voting-configuration-sql-server.md?view=sql-server-ver16)  
```

**Comment by Jak-MS** [comment]
@MashaMSFT 
- Can you review this PR?

IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/sql-server/failover-clusters/windows/wsfc-disaster-recovery-through-forced-quorum-sql-server.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ ms.author: mathoma
 > [!WARNING]  
 >  The user should be well-informed on the concepts and interactions of Windows Server Failover Clustering, WSFC Quorum Models, [!INCLUDE[ssNoVersion](../../../includes/ssnoversion-md.md)], and the environment's specific deployment configuration.  
 >   
->  For more information, see:  [Windows Server Failover Clustering (WSFC) with SQL Server](https://msdn.microsoft.com/library/hh270278\(v=SQL.110\).aspx), [WSFC Quorum Modes and Voting Configuration (SQL Server)](https://msdn.microsoft.com/library/hh270280\(v=SQL.110\).aspx)  
+>  For more information, see:  [Windows Server Failover Clustering (WSFC) with SQL Server](windows-server-failover-clustering-wsfc-with-sql-server.md?view=sql-server-ver16), [WSFC Quorum Modes and Voting Configuration (SQL Server)](wsfc-quorum-modes-and-voting-configuration-sql-server.md?view=sql-server-ver16)
```
Comment (rwestMSFT) [inline docs/sql-server/failover-clusters/windows/wsfc-disaster-recovery-through-forced-quorum-sql-server.md:40]:
```suggestion
>  For more information, see [Windows Server Failover Clustering (WSFC) with SQL Server](windows-server-failover-clustering-wsfc-with-sql-server.md), and [WSFC Quorum Modes and Voting Configuration (SQL Server)](wsfc-quorum-modes-and-voting-configuration-sql-server.md).
```

**Comment by MashaMSFT** [comment]
Hi @0x7FFFFFFFFFFFFFFF , thanks so much for your submission! My apologies for the delay, we've been working through a long backlog of items. I've pulled your PR into the private repository and processed so I'll go ahead and close out this PR but you will still show up as a contributor to the doc once the change goes live. Please feel free to comment should you need anything further. 

Thanks again for improving our docs and I hope you have a wonderful rest of your day!
Masha from the SQL Docs team

**Pull request closed without merging**